src/System.Management.Automation/engine/SessionStateLocationAPIs.cs

```diff
@@ -266,6 +267,16 @@ internal PathInfo SetLocation(string path, CmdletProviderContext context)
                     PSDriveInfo newWorkingDrive = GetDrive(driveName);
                     CurrentDrive = newWorkingDrive;
 
+                    // If the path is simply a colon-terminated drive,
+                    // not a slash-terminated path to the root of a drive,
+                    // set the path to the current working directory of that drive.
+
```

Comment:
A small style point -- I'd remove the newline here so the comment doesn't float.

Having looked at some of the older parts of this codebase, my experience is that comments that aren't glued to code have a tendency to float off into limbo (so that a couple of years down the track, an old comment will sit above a wildly different block of code with nobody certain about what it's referring to).

Reply:
Thanks for the update! I'll adjust this later tonight.

Reply:
Newlines have been removed from the floating comments.